media-plugins/vdr-graphlcd/vdr-graphlcd-1.0.0.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="vdr-plugin-graphlcd"
+MY_P="${MY_PN}-${PV}"
+
+inherit vdr-plugin-2
+
+DESCRIPTION="VDR Plugin: support output on Graphical LCD "
+HOMEPAGE="https://projects.vdr-developer.org/projects/graphlcd"
+SRC_URI="https://projects.vdr-developer.org/git/${MY_PN}.git/snapshot/${MY_P}.tar.bz2"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-2"
+SLOT="0"
+
+DEPEND=">=media-video/vdr-1.6
+	>=app-misc/graphlcd-base-${PV}"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${MY_P}"
+
+pkg_setup() {
+	vdr-plugin-2_pkg_setup
+
+	if ! getent group lp | grep -q vdr; then
+		echo
+		einfo "Add user 'vdr' to group 'lp' for full user access to parport device"
+		echo
+		elog "User vdr added to group lp"
+		gpasswd -a vdr lp
```

Comment:
Can't this be done with user.eclass? `|| die` otherwise.

Reply:
Nope, `user.eclass` seems not to provied this feature. Added `|| die`.